tests/integrate_test/cita_features_test.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+set -e
+
+if [[ $(uname) == 'Darwin' ]]
+then
+    SOURCE_DIR=$(realpath "$(dirname "$0")"/../..)
+else
+    SOURCE_DIR=$(readlink -f "$(dirname "$0")"/../..)
+fi
+BINARY_DIR=${SOURCE_DIR}/target/install
+
+test_fee_back() {
+    local version=$1
+    cd ./scripts/txtool/txtool
+    python3 "${SOURCE_DIR}"/tests/integrate_test/test_fee_back.py --version="$version"
+    cd ../../..
+}
+
+test_perm_denied() {
+    local version=$1
+    cd ./scripts/txtool/txtool
+    python3 "${SOURCE_DIR}"/tests/integrate_test/test_perm_denied.py --version="$version"
+    cd ../../..
+}
+
+main() {
+    echo -n "0) prepare  ...  "
+    # shellcheck source=/dev/null
+    . ${SOURCE_DIR}/tests/integrate_test/util.sh
+    cd "${BINARY_DIR}"
+    echo "DONE"
+
+    echo -n "1) generate config  ...  "
+    create_config \
+        --contract_arguments "SysConfig.checkFeeBackPlatform=true" \
+        --contract_arguments "SysConfig.checkCreateContractPermission=true" \
+        --contract_arguments "SysConfig.economicalModel=1" \
+        --contract_arguments "VersionManager.version=0" \
```

Comment:
What about version 1 and 2?
Why do not use version 2 directly?

Reply:
I just copy from `cita_charge_mode`, version=2 is ok.

Reply:
ok.
`cita_charge_mode` testes both version and version 1. Your copy seems miss version 1.

Reply:
I remove version config at start, just use latest version 2